src/status_im/ui/screens/routing/core.cljs

```diff
@@ -97,11 +113,13 @@
                       (get style :padding-vertical))})))
 
 (defn presentation-type [{:keys [transition] :as opts}]
-  (if (and platform/ios? (= transition :presentation-ios))
+  (if (and platform/ios? (= transition :presentation-modal))
```

Comment:
[`cond`](https://clojuredocs.org/clojure.core/cond) is more appropriate than `if`.